src/Listener/SendNotificationWhenUserIsSuspended.php

```diff
@@ -79,7 +79,7 @@ public function whenUserWasUnsuspended(UserWasUnsuspended $event)
      */
     public function sync(User $user, User $actor, array $recipients)
     {
-        if ($user->suspend_until > new DateTime()) {
+        if (isset($user->suspend_until) && $user->suspend_until > new DateTime()) {
```

Comment:
I'd prefer to see this line refactored. First off Eloquent returns `null` for any property on a Model that does not exist. So it's safe to do `if ($user->suspend_until`, additionally the `suspend_until` property if set will be a Carbon object. As such you should use its API instead of using a PHP native DateTime object. Hence I'd suggest using:

```php
if ($user->suspend_until && $user->suspend_until->isFuture()) {
```